_spinnaker/install.md

```diff
@@ -49,6 +49,7 @@ SUBCOMMANDS
   init
     Runs Armory installer
 ```
+All standard [halyard commands](https://www.spinnaker.io/reference/halyard/) are also available.
```

Comment:
what do you think about keeping the same language:
> From there, you can issue all your [halyard commands](https://www.spinnaker.io/reference/halyard/).

Reply:
Sounds good to me, I'll make the change